packages/backend/src/logger/config.d.ts

```diff
@@ -0,0 +1,79 @@
+export interface Config {
+  /**
+   * Configuration for the audit logging
+   * @visibility frontend
+   */
+  auditLog?: {
+    /**
+     * Configures whether to log audit logs to the console
+     * Default: true
+     * @visibility frontend
+     */
+    logToConsole: boolean;
```

Comment:
```suggestion
    logToConsole: {
      enabled: boolean;
    };
```

Is adopting a standardized method for enabling options a good idea?

Reply:
Hmm maybe, I mainly just did a `logToConsole` config since I assumed we wouldn't need any additional configurations for console audit logs?